guvnor-webapp/src/main/java/org/guvnor/client/perspectives/M2RepoPerspective.java

```diff
@@ -26,7 +26,8 @@
 import com.google.gwt.user.client.ui.PopupPanel;
```

Comment:
What do you think about  this file is it correct changes?

P.S. There is another 5 files with using old UploadForm:

kie-wb-distributions/kie-drools-wb/kie-drools-wb-webapp/src/main/java/org/kie/workbench/drools/client/perspectives/M2RepoPerspective.java
kie-wb-distributions/kie-wb/kie-wb-webapp/src/main/java/org/kie/workbench/client/perspectives/M2RepoPerspective.java
optaplanner-wb/optaplanner-wb-webapp/src/main/java/org/optaplanner/workbench/client/perspectives/M2RepoPerspective.java
experiments/guvnor/guvnor-webapp/src/main/java/org/guvnor/client/perspectives/M2RepoPerspective.java
drools-wb/drools-wb-webapp/src/main/java/org/drools/workbench/client/perspectives/M2RepoPerspective.java

So before merge we have to change it also.